scripts-docker/run

```diff
@@ -3,11 +3,13 @@
 echo "Running pmo-api ..."
 docker run \
 -p 7070:7070 \
+--name donejs-api \
```

Comment:
@Mattchewone Can you name these `place-my-order-api` and `place-my-order` ?

Reply:
Consider it done